config_archive_mainnet.yaml

```diff
@@ -16,15 +16,14 @@ chain:
   chainDBPath: "/var/iotex-archive/data/chain.db"
   trieDBPatchFile: "/var/iotex-archive/data/trie.db.patch"
   trieDBPath: "/var/iotex-archive/data/archive.db"
+  stakingPatchDir: "/var/iotex-archive/data",
   indexDBPath: "/var/iotex-archive/data/index.db"
   blobStoreDBPath: "/var/iotex-archive/data/blob.db"
   bloomfilterIndexDBPath: "/var/iotex-archive/data/bloomfilter.index.db"
   candidateIndexDBPath: "/var/iotex-archive/data/candidate.index.db"
   stakingIndexDBPath: "/var/iotex-archive/data/staking.index.db"
   contractStakingIndexDBPath: "/var/iotex-archive/data/contractstaking.index.db"
-  enableVersionedStateDB: true
-  versionedNamespaces: ["Account", "Contract"]
-  versionedMetadata: "Meta"
+  enableArchiveMode: true
```

Comment:
per code review, these should not be user option, now use the existing `enableArchiveMode` flag